tools/vagrant/Vagrantfile

```diff
@@ -6,7 +6,7 @@ BOX = "ubuntu/trusty64-2"
 BOX_URL =  "https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-amd64-vagrant-disk1.box"
 BOX_MEMORY = ENV['BOX_MEMORY'] || '4096'
 BOX_CPUS = ENV['BOX_CPUS'] || '8'
-MACHINE_IP = ENV['MACHINE_IP'] || '192.168.33.13'
+MACHINE_IP = ENV['MACHINE_IP'] || '192.168.56.111'
```

Comment:
@paulcastro @psuter  this IP Address `192.168.33.13` is reference in README.md, if there is no good reason I would say do not change it, as folks might already have this IP scripted in their workflow

https://github.com/openwhisk/openwhisk/blob/437f2ef1288839393b9b026f13b6665edcbd3bd2/README.md#L287
https://github.com/openwhisk/openwhisk/blob/437f2ef1288839393b9b026f13b6665edcbd3bd2/README.md#L290